test/metabase/api/activity_test.clj

```diff
@@ -47,7 +47,7 @@
              ;; view log entries look for the `:actor_id` in the item being viewed to set that view's :user_id
              (assoc item :actor_id (mt/user->id :crowberto))))
           (testing "most_recently_viewed_dashboard endpoint shows the current user's most recently viewed dashboard."
-            (is (= dash-3 #_dash-2 ;; TODO: this should be dash-2, because dash-3 is archived
+            (is (= (assoc dash-3 :collection nil) #_dash-2 ;; TODO: this should be dash-2, because dash-3 is archived
```

Comment:
Can you explain why this is nil to me? In the `most-recently-viewed-dashboard-views-include-colllection-test` it is clearly non-nil so I'm just wondering what's the difference here

Reply:
because `dash-3` here is in the root (aka Our analytics) collection, and it's a virtual collection, which is why it's nil here.

the dashboard in `most-recently-viewed-dashboard-views-include-colllection-test` is placed under a collection.

Reply:
Makes sense, thanks